generator/ServiceClientGeneratorLib/Customizations.cs

```diff
@@ -600,6 +600,22 @@ public List<string> NoArgOverloads
             }
         }
 
+        /// <summary>
+        /// For Lists of enums, the generator automatically treats the enum as a string via passing in true to "treatEnumAsString".
+        /// This customization overrides that so that the enum is treated as an enum. 
+        ///     "treatEnumsAsEnums":{
```

Comment:
You need to update the example to match the rename to `OverrideTreatEnumsAsString`